Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -214,22 +221,41 @@ - (void)styleSearchBar {
     }
 }
 
-- (void)updateRowHeight:(id)sender {
-    
-    BOOL condensedNoteList = [[NSUserDefaults standardUserDefaults] boolForKey:SPCondensedNoteListPref];
-    
+- (void)refreshTableViewCellStyles {
+
+    for (UIView *subview in self.tableView.subviews) {
```

Comment:
Would `[self.tableView visibleCells]` work better here? 

I'm wondering if would it make sense to also call refreshStyles from the cell's prepareForReuse method? I'm thinking about cells that are not currently visible / subviews but have previously been dequeued and might still be reused.

Reply:
You know what... doing this in `prepareForReuse` is much cleaner, honestly. I took this road for performance reasons, **but** we're not using iPhone 3gs's anymore!

As per `self.tableView.visibleCells`, i'm afraid it wasn't cutting it. Scenario was:

1. Launch the app
2. Toggle accessibility, and increase font sizes
3. Back to Simplenote
4. Toggle Dark Mode

In this scenario, `visibleCells` wasn't retrieving **every** cell, that's why i've opted for `.subviews`. But nevertheless, there is no warranty that all the cells will be effectivelly subviews.

I'll move this to `prepareForReuse`, thank you!!